ballerina/modules/mock/openapi_service.bal

```diff
@@ -0,0 +1,478 @@
+// Copyright (c) 2023, WSO2 LLC. (http://www.wso2.com) All Rights Reserved.
+//
+// WSO2 LLC. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+import ballerina/http;
+
+listener http:Listener ep0 = new (9090, config = {host: "localhost"});
+
+service /calendar/v3 on ep0 {
```

Comment:
we can initilize the listener inline if we are not using ep0 elsewhere

Reply:
Fixed in [3253004](https://github.com/ballerina-platform/module-ballerinax-googleapis.calendar/pull/138/commits/3253004980ab8afc4246ab403f4cb732c5841f55)